docs/hub/repositories-best-practices.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Getting Started with Repositories
+---
+
+<h1>Best practices with repositories</h1>
+
+Here are some additional best practices to help you get the most out of your repository.
```

Comment:
Could you please add a section named "Handling multiple experiments"  with a large TODO? (I can help with that in a follow-up PR from my side) 

I think we can add things from https://github.com/huggingface/huggingface_hub/issues/769 and specially from https://github.com/huggingface/hub-docs/issues/53